docs/application/native/guides/security/privacy-related-permissions.md

```diff
@@ -18,7 +18,7 @@ The main features of the Privacy Privilege Manager API include:
 For a list of privacy-related privileges, see [Security and API Privileges](../../tutorials/details/sec-privileges.md).
 
 > [!NOTE]
-> Since Tizen 5.5, if the caller app component type is UI app or component based app with frame component, then, the pop-ups by `ppm_request_permission()` and `ppm_request_permissions()` are launched as `group mode` with the caller app.
+> Since Tizen 5.5, if the caller app component type is UI app then the pop-ups by `ppm_request_permission()` and `ppm_request_permissions()` are launched as `group mode` with the caller app.
```

Comment:
How about below little changes?

```suggestion
> Since Tizen 5.5, if the caller application component type is UI application, then the pop-ups by `ppm_request_permission()` and `ppm_request_permissions()` are launched as `group mode` with the caller application.
```

Reply:
@mijin85cho yes looks fine.